src/zenml/zen_server/deploy/helm/values.yaml

```diff
@@ -956,6 +1016,35 @@ zenml:
       generateCerts: false
       secretName: zenml-tls-certs
 
+  # Configuration for custom CA certificates.
+  certificates:
+
+    # Direct injection of CA certificates in PEM format
+    customCAs: []
```

Comment:
naming this customCAs is a bit redundant as this block is all about custom CAs anyways. having this variable here and then a secretrefs next doesn't clearly tell you that they both point to custom CAs.
maybe we can use `inline` ?